scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/soic.yaml

```diff
@@ -349,6 +349,64 @@ SOIC-8-1EP_3.9x4.9mm_P1.27mm_EP2.514x3.2mm:
     # bottom_pad_size:
     paste_avoid_via: False
 
+    
+       
+SOIC-8-1EP_3.9x4.9mm_P1.27mm_EP2.62x3.51mm:
+  size_source: 'https://www.monolithicpower.com/en/documentview/productdocument/index/version/2/document_type/Datasheet/lang/en/sku/MP2303A/document_id/494#page=14'
+  body_size_x:
+    nominal: 3.9
```

Comment:
Use the exact dimensions and dimension format from the datasheet. This helps in checking. So instead of nominal and tolerance, use min and max like the datasheet. This applies to all dimensions.